README.md

```diff
@@ -26,45 +26,25 @@
 
 
 ## Introduction
+Pandora is a tool for bacterial genome analysis using a pangenome reference graph (PanRG). It allows gene presence/absence detection and genotyping of SNPs, indels and longer variants in one or a number of samples. Pandora works with Illumina or Nanopore data. 
 
-Pandora is a tool for bacterial genome analysis using a pangenome
-reference graph (PanRG). It allows gene presence/absence detection and
-genotyping of SNPs, indels and longer variants in one or a number of
-samples. Pandora works with Illumina or Nanopore data. Core ideas behind
-the method are:
-- new genomes look like recombinants (plus mutations) of things seen
-  before
-- we should be analysing nucleotide-level variation everywhere, not just
-  in core genes
-- arbitrary single reference genomes are unnatural and limit comparisons
-  of diverse sets of genomes
-
-The pangenome reference graph (PanRG) is a collection of 'floating'
+The PanRG is a collection of 'floating'
 local graphs, each representing some orthologous region of interest
 (e.g. genes, mobile elements or intergenic regions). See
 https://github.com/rmcolq/make_prg for a pipeline which can construct
-these PRGs from a set of aligned sequence files.
+these PanRGs from a set of aligned sequence files.
 
 Pandora can do the following for a single sample (read dataset):
-- Output inferred mosaic of reference sequences for loci (eg genes) from
-  the PanRG which are present
-- Output a VCF showing the variation found within these loci, with
-  respect to any reference path in the PRG.
-
-Soon, in a galaxy not so far away, it will allow:
-- discovery of new variation not in the PRG
+- Output inferred mosaic of reference sequences for loci (eg genes) from the PanRG which are present;
+- Output a VCF showing the variation found within these loci, with respect to any reference path in the PRG;
+- Discovery of new variation not in the PanRG.
 
 For a collection of samples, it can:
-- Output a matrix showing inferred copy-number of each locus in each
-  sample genome
-- Output a multisample pangenome VCF showing how including genotype
-  calls for each sample in each of the loci
-- Output one VCF per orthologous-chunk, showing how samples which
-  contained this chunk differed in their gene sequence. Variation is
-  shown with respect to the most informative recombinant path in the
-  PRG.
+- Output a matrix showing inferred copy-number of each locus in each sample genome;
```

Comment:
@iqbal-lab  @mbhall88 : just tagging you to inform that I am changing this from "copy-number" to "presence-absence" ?